src/main/views/macro/support.njk

```diff
@@ -6,15 +6,15 @@
 {% endmacro%}
 
 {% macro _emailWidget(email = supportEmailAddress) %}
-    <h2 class="heading-medium" style="margin-bottom:5px; margin-top: 10px;">{{ t('Email') }}</h2>
-    <pre><a href="mailto:mailto:{{t(email)}}" target="_blank">{{ t(email)}}</a></pre>
-    <pre>{{ t("We'll get back to you within 10 days.") }}</pre>
+    <h2 class="heading-medium" style="margin-bottom:5px; margin-top: 1.05263em;">{{ t('Email') }}</h2>
+    <pre style="margin-top: 0.26316em"><a href="mailto:{{t(email)}}" target="_blank">{{ t(email)}}</a></pre>
```

Comment:
please either use existing classes `form-group-related` or create a new class. Inline styles should be avoided when possible

Reply:
chawed code to use span